common/src/main/java/org/opensearch/ml/common/dataset/TextSimilarityInputDataSet.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2023 Aryn
```

Comment:
as I understand, all this does is protect us on the off-chance that AWS does what Elastic did a few years ago and switches out the license or something. This is coming from Mehul, who oversaw that whole transition (elastic fork -> opensearch).

Reply:
> protect us on the off-chance that AWS does what Elastic did a few years ago and switches out the license or something

No need to worry that actually, if you check Elastic, community can still use the open source version even they changed license, otherwise we can't do "elastic fork -> opensearch".  If you worry this part, I think we should suggest not adding Aryn to license header. What if some other company make a change to this file, they may also prefer to add their license. Then in future how can Aryn tell which company holds the license for this file? It looks not reasonable that not allowing other guys modifying your code, right?  

I guess the other reason for adding Aryn is to show the credit of this feature. We are planning to build some way to show attribution/appreciation for features from community contributors. For example, we can maintain a contribution list file like this

| Feature      | Description |  Contributor |
| ----------- | ----------- | ----------- |
| Conversation search      | This feature will...       | Austin (from Aryn.ai) Henry (from Aryn.ai) |

Any suggestion?

Reply:
we already decided this with @sean-zheng-amazon and @mashah

Reply:
This was decided prior to the 2.10 release as part of a PR that we contributed.  I believe the initial discussion was done in that PR, but was later concluded over a call.